modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
+
+.Procedure
+Customize the cluster role options of all projects that are shown in the *Project Access* tab:
```

Comment:
Above, you say to customize the cluster role's name. Here, you say to customize the cluster role's options. Which is it? The title says "roles" so perhaps here you can say "role" instead of "option" for consistency? Unless they are two different things?

Also, you sometimes use possessive: cluster role's name. 
Other times, you don't use a possessive: cluster role options. 

Pick one to be consistent. I would go with whichever case is more common in openshift docs. If it helps, grammatically, it would be possessive since both the name and options belong to the cluster role.

Reply:
Good point, I'll keep this suggestion while rephrasing the content.

Reply:
@jc-berger Thank you, Jake, for looking into this.
It was one of my concerns too: the introductory paragraph seems a bit confusing to me since it describes some fine-tuning option (customizing roles names) while it's not this module's main topic. I hope you could help us solve this, thanks.